.docker/app/Dockerfile

```diff
@@ -6,13 +6,13 @@ COPY . .
 
 USER root
 
-ARG RAILS_MASTER_KEY \
+ARG RAILS_MASTER_KEY=MANIFOLD_MASTER_KEY \
```

Comment:
I think we need to include a $ in front of MANIFOLD_MASTER_KEY

Reply:
done